tests/lib/packaging/build-arch.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+snapd_dir=$1
+user=$2
+build_dir=$3
+
+cd "$snapd_dir"
+
+version=
+for file in "$build_dir"/snapd_*.vendor.tar.xz; do
+        version="${file##*/snapd_}"
+        version="${version%.vendor.tar.xz}"
+        break
+done
+cp -av packaging/arch/* "$build_dir"
+sed -i -e "s/pkgver=.*/pkgver=$version/" "$build_dir"/PKGBUILD
```

Comment:
rather than here (and so the $version guessing shenanigans are no longer needed)